web/src/index.html

```diff
@@ -9,6 +9,6 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="app"></div>
-    <script src='app.js'></script>
+    <script src='/app.js'></script>
```

Comment:
i don't understand why this is needed, but it also doesn't seem hurtful

Reply:
Ah, it's because of the nested routes I had in the earlier version.  Not necessary for now, but when there were routes like `/expenses/list`, if you reloaded the page while at that route, the it would fail because it couldn't find `/expenses/app.js`.  So I made the path absolute.  😛